### ISSUE TITLE
[EARTH SEARCH] Broken JSON

### ISSUE THREAD

**Issue by remi-braun** [CS-SI/eodag#382]
A search with Earth Search returns a broken JSON:

![2022-01-07_16h44_33](https://user-images.githubusercontent.com/67311115/148568502-38081063-b318-40a6-9497-10fe1b62d683.png)

```
2022-01-07 16:39:22 - [INFO] - Querying S2_MSI_L2A products.
2022-01-07 16:39:24 - [INFO] - Searching product type 'S2_MSI_L2A' on provider: earth_search
2022-01-07 16:39:24 - [INFO] - Iterate search over multiple pages: page #1
2022-01-07 16:39:24 - [INFO] - Sending search request: https://earth-search.aws.element84.com/v0/search?datetime=2021-08-04T00:00:00.000Z/2021-08-05T00:00:00.000Z&bbox=[-72.5865387212294,18.480703895829237,-72.5865387212294,18.480703895829237]&collections=["sentinel-s2-l2a"]&limit=500&page=1
2022-01-07 16:39:25 - [ERROR] - Skipping error while searching for earth_search StacSearch instance: 
Traceback (most recent call last):
  File "C:\Users\rbraun\Anaconda3\envs\eodownload\lib\site-packages\eodag\plugins\search\qssearch.py", line 716, in _request
    response.raise_for_status()
  File "C:\Users\rbraun\Anaconda3\envs\eodownload\lib\site-packages\requests\models.py", line 953, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://earth-search.aws.element84.com/v0/search?datetime=2021-08-04T00:00:00.000Z/2021-08-05T00:00:00.000Z&bbox=%5B-72.5865387212294,18.480703895829237,-72.5865387212294,18.480703895829237%5D&collections=%5B%22sentinel-s2-l2a%22%5D&limit=500&page=1
```

`https://earth-search.aws.element84.com/v0/search?datetime=2021-08-04T00:00:00.000Z/2021-08-05T00:00:00.000Z&bbox=[-72.5865387212294,18.480703895829237,-72.5865387212294,18.480703895829237]&collections=["sentinel-s2-l2a"]&limit=500&page=1` does not work :
![2022-01-07_16h50_31](https://user-images.githubusercontent.com/67311115/148569359-c1a769c3-6f35-44b7-bebf-9bdeeb889731.png)

However, 
`https://earth-search.aws.element84.com/v0/search?datetime=2021-08-04T00:00:00.000Z/2021-08-05T00:00:00.000Z&collections=["sentinel-s2-l2a"]&limit=500&page=1`
or
`https://earth-search.aws.element84.com/v0/search?datetime=2021-08-04T00:00:00.000Z/2021-08-05T00:00:00.000Z&bbox=[-72.5865387212294,18.480703895829237,-72.5865387212294,18.480703895829237]&ollections=["sentinel-s2-l2a"]&page=1`
or even
`https://earth-search.aws.element84.com/v0/search?datetime=2021-08-04T00:00:00.000Z/2021-08-05T00:00:00.000Z&bbox=[-72.5865387212294,18.480703895829237,-72.5865387212294,18.480703895829237]&collections=[%22sentinel-s2-l1c%22]&limit=500&page=1`
are working. 😓 

Environment:

    Python version: Python 3.7.12
    EODAG version: eodag (Earth Observation Data Access Gateway): version 2.3.4

**Comment by sbrunato** [comment]
Hello @remi-braun , I notice that the search geometry is a point : `longitude_max == longitude_min` and `latitude_max == latitude_min`.
And your search requests that work do not include this bbox or are returning 0 item:
- Slightly changing the bbox works: https://earth-search.aws.element84.com/v0/search?datetime=2021-08-04T00:00:00.000Z/2021-08-05T00:00:00.000Z&bbox=[-72.9865387212294,18.680703895829237,-72.5865387212294,18.480703895829237]&collections=[%22sentinel-s2-l2a%22]&limit=500&page=1
- Using `"POINT(-72.5865387212294 18.480703895829237)"` as search parameter work running `eodag.git@develop` (#363 which enables advanced geometry usage will be included in the next incoming release)

**Comment by remi-braun** [comment]
- For the first point, the query works with L1C but not with L2A (and the query with this bbox works for the other providers), so this is really strange. But it is maybe not an EODAG issue...
- And I will test the POINT query when the newt release comes out 😉 (can't wait !)

**Comment by sbrunato** [comment]
> For the first point, the query works with L1C but not with L2A (and the query with this bbox works for the other providers), so this is really strange. But it is maybe not an EODAG issue...

That's true but the search returns 0 items. Changing dates also breaks the request https://earth-search.aws.element84.com/v0/search?datetime=2020-08-04T00:00:00.000Z/2021-08-05T00:00:00.000Z&bbox=[-72.5865387212294,18.480703895829237,-72.5865387212294,18.480703895829237]&collections=[%22sentinel-s2-l1c%22]&limit=500&page=1

**Comment by remi-braun** [comment]
Is it a way to handle this exception in eodag ? In order not to crash unexpectedly 😅

**Comment by sbrunato** [comment]
fixed in #275
